Real_world_examples/Radar_water_detection.ipynb

```diff
@@ -2309,11 +2309,663 @@
    "metadata": {},
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # Search the STAC Catalog
</pre><p>No need for this; only required for datacube access</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GeoscienceAustralia/dea-notebooks/pull/1263/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='77'/>